rxpager/build.gradle

```diff
@@ -1,26 +1,21 @@
-apply plugin: 'android-sdk-manager'
 apply plugin: 'com.android.library'
-apply from: '../quality.gradle'
-apply plugin: 'me.tatarka.retrolambda'
 apply plugin: 'com.novoda.bintray-release'
 
-retrolambda {
-  javaVersion JavaVersion.VERSION_1_7
-}
-
 group = 'com.schibstedspain.android'
-version = '1.0.0'
+version = '2.0.0'
 
 android {
-  compileSdkVersion 24
-  buildToolsVersion "24.0.1"
+  compileSdkVersion 27
 
   defaultConfig {
-    minSdkVersion 9
-    targetSdkVersion 24
+    minSdkVersion 14
```

Comment:
Why minSdk 14? Do we really need it?
I don't think it matters a lot but as a library keeping it as low as possible is always desired.

Reply:
I changed it because the IDE complained about it, also it suggested me to raise it to version 14. So I decided to do it. Anyway, who is actually using version under API 14? Does it have sense to maintain that version?

Reply:
It's just to be sincere.
This module doesn't have any dependencies that require minSdk 14, at least that I know.

When you use for example the support libs it requires minSdk 14.

Having the library with the lowest minSdk allows more people to use it. 14 is a reasonable minSdk, but if it's free... having minSdk 1 would be desirable.